proxysql/build-binary-proxysql

```diff
@@ -36,6 +45,7 @@ sudo wget https://dl.google.com/go/go1.17.7.linux-amd64.tar.gz
 # Hence using the default $PATH that is available to every new session (both parent and child script)
 sudo tar -C /usr/local/ -xzf go1.17.7.linux-amd64.tar.gz
 sudo cp /usr/local/go/bin/go /usr/local/bin/go
+sudo ln -s /usr/local/go/bin/go /usr/bin/go
```

Comment:
Why are we creating this softlink when we already have go installed at
/usr/local/bin and this path exists in $PATH already

Reply:
I have tested on the same docker image that is used for building.
<img width="737" alt="Screenshot 2022-06-16 at 17 58 08" src="https://user-images.githubusercontent.com/25184101/174099447-02cc8c4a-14e3-4dd9-8e02-12fb6f0f944e.png">

Reply:
You can retain the softlinks